lib/src/page_list_viewport_gestures.dart

```diff
@@ -242,6 +253,8 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
   void _startMomentum() {
     PageListViewportLogs.pagesListGestures.fine(() => "Starting momentum...");
     final velocity = _panAndScaleVelocityTracker.velocity;
+    final momentumSimInitialVelocitySclar = _panAndScaleVelocityTracker.momentumSimInitialVelocitySclar;
+    final dragIncreaseScalar = _panAndScaleVelocityTracker.dragIncreaseScalar;
```

Comment:
Is there a more descriptive term than `dragIncreaseScalar`? As a reader, I have no idea what that means as I read it. I can imagine many possible meanings for that.

Reply:
kIncreaseScalar terminology is my attempt to mean that it should multiply the value k.
 So k = k*kIncreaseScalar.
 How would you name it?
 Hopefully new comments will clarify better what I mean.

Reply:
Ozzie and I call the simulation that gets launched after a fast panning event a momentum simulation. That simulation gets an initial velocity, the first order fluid drag coefficient and a static friction coefficient. This simulation is launched immediately after the user lifts the finger after a fast pan. If they pan repeatedly, simulations would be launched with an ever higher initial velocity. This increase in initial velocity is defined by the scalar(scaler). A thing that scales. Maybe we can call it simulationInitialVelocityMagnifier 
 / amplifier?

Reply:
I agree with you that momentum is a confusing name. Would you be okay with me renaming it into a kinematicScrollSimulation?